CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -1609,8 +1606,6 @@ local function CreateNewSlider(numOfSteps, startStep, minStep)
 	local stepXScale = 1 / steps
 	stepXSize = 0
 
-	local stepsAspectConstraints = {}
```

Comment:
Maybe you're looking at UtilityOld.lua, @hockeychaos. I was updating these aspect constraints on viewport resize, but I decided that was a bad idea and forgot to remove this part. My bad!